cmd/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"log"
+	"timetracker"
+)
+
+func main() {
+
+	s := timetracker.NewServer()
```

Comment:
Great! A very small and clear `main` function, with all the behaviour located in your engine package. Well done.